msteams-platform/tabs/how-to/authentication/auth-tab-aad.md

```diff
@@ -46,14 +46,14 @@ Authentication flow should be triggered by a user action. You shouldn't open the
 
 Add a button to your configuration or content page to enable the user to sign in when needed. This can be done in the tab [configuration](~/tabs/how-to/create-tab-pages/configuration-page.md) page or any [content](~/tabs/how-to/create-tab-pages/content-page.md) page.
 
-Azure AD, like most identity providers, doesn't allow its content to be placed in an `iframe`. This means that you'll need to add a pop-up page to host the identity provider. In the following example, this page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the TeamsJS library to launch this page when the button is selected.
+Azure AD, like most identity providers, doesn't allow its content to be placed in an `iframe`. This means you need to add a page to host the identity provider that is displayed inside a pop-up window by Teams client. In the following example, the page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the TeamsJS library to launch this page when the button is selected.
```

Comment:
This comment is for line no 13 and 14. 
13
The first usage of Teams app should be Microsoft Teams app:
There are many services that you may want to consume inside your Microsoft Teams app, and most of those services require authentication and authorization to get access to the service. Services includes Facebook, Twitter, and Teams.
14
The first usage of Azure AD should be Azure Active Directory (Azure AD). For the rest of the doc, you can use Azure AD.
Teams user profile information is stored in Azure Active Directory (Azure AD) using Microsoft Graph and this article will focus on authentication using Azure AD to get access to this information.